venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,718 @@
+                "required": True,
+                "order": 10
+                },
+            "paper_type": {
+                "description": "Select if your paper is short or long",
+                "value-radio": [
+                    "short",
+                    "long"
+                ],
+                "required": True,
+                "order": 11
+                },
+            "track": {
+                "description": "Select the track that best fits your submission.",
+                "value-radio": [
+                    "Ethics and NLP",
+                    "Linguistic theories, Cognitive Modeling and Psycholinguistics",
+                    "Machine Learning for NLP",
+                    "Phonology, Morphology and Word Segmentation",
+                    "Resources and Evaluation",
+                    "Semantics: Lexical",
+                    "Semantics: Sentence-level Semantics, Textual Inference and Other areas",
+                    "Syntax: Tagging, Chunking and Parsing",
+                    "Information Extraction",
+                    "Computational Social Science and Cultural Analytics",
+                    "Information Retrieval and Text Mining",
+                    "Interpretability and Analysis of Models for NLP",
+                    "Machine Translation and Multilinguality",
+                    "NLP Applications",
+                    "Question Answering",
+                    "Dialogue and Interactive Systems",
+                    "Discourse and Pragmatics",
+                    "Generation",
+                    "Language Grounding to Vision, Robotics and Beyond",
+                    "Sentiment Analysis, Stylistic Analysis, and Argument Mining",
+                    "Speech and Multimodality",
+                    "Summarization",
+                    "Special Theme"
+                ],
+                "required": True,
+                "order": 12
+                },
+            "comment": {
+                "description": "Comment to Senior Area Chairs (500 words)",
+                "order": 13,
+                "value-regex": "[\\S\\s]{0,500}",
+                "required": False
+                }
+        }
+    }
+)
+
+official_review = openreview.Invitation(
+    id = 'aclweb.org/ACL/2022/Conference/-/Official_Review',
+    signatures = [
+        'aclweb.org/ACL/2022/Conference'
+    ],
+    readers = [
+        'everyone'
+    ],
+    writers = [
+        'aclweb.org/ACL/2022/Conference'
+    ],
+    reply = {
+        "readers": {
+            "values-copied":[
+                'aclweb.org/ACL/2022/Conference',
+                'aclweb.org/ACL/2022/Conference/Program_Chairs'
+        ]},
+        "writers": {
+            "values-copied":[
+                'aclweb.org/ACL/2022/Conference'
+        ]},
+        "signatures": {
+            "values-copied": [
+                'aclweb.org/ACL/2022/Conference'
+            ]
+        },
+        "content":{
+            "confidence": {
```

Comment:
I wonder if the review form is the same for all the ARR iterations.

Reply:
I merged all of the different ARR reviews into the invitation and set all of the content as false with the exception of the 'metareview'/'review' fields

Reply:
great, thank you!